USAGE.md

```diff
@@ -0,0 +1,107 @@
+# Amazon Alexa Smart Home skill for openHAB 2
+
+Amazon certified openHAB2 as a [smarthome skill for Alexa](https://www.amazon.com/openHAB-Foundation/dp/B01MTY7Z5L).
+This skill allows you to connect your openHAB setup through the myopenHAB.org cloud service to Amazon Alexa.
+With this skill you can control items that respond to on/off and numeric commands as well as limited support for thermostats.
```

Comment:
the second part doesn't match the sentence semantic.

Reply:
I also copied that from the announcement. I am not sure hwat you want me to change. Give me an advice please.. TY :)

Reply:
"With this skill you can control ... as well as limited support for thermostats." -> The sentence is not correct :)

Reply:
Ill seperate in two sentences.